phpstan.neon.dist

```diff
@@ -30,34 +30,30 @@ parameters:
 			path: plugins/dominant-color-images/*.php
 
 		# The following ignored errors are all for tests. They are likely excessively strict and do not warrant being fixed.
-		# TODO: Use error identifiers instead of messages once PHPStan 1.11 is out: <https://github.com/WordPress/performance/issues/1209>.
 		-
-			message: '/Parameter #\d.+given./'
+			identifier: argument.type
```

Comment:
Note: I tried originally to use `identifier: argument.type` with the ignored error above in `dominant-color-images`. However, it seems you can only use an `identifier` once in the `ignoreErrors`, or else the ignore is ignored (such _ignorance_!) But it is better anyway to use the `message` above since it is explicitly about `GdImage|resource` being used.